config.yaml

```diff
@@ -0,0 +1,26 @@
+projectName: platform
+image: jembi/platform:latest
+logPath: /tmp/logs
+
+packages:
+  - interoperability-layer-openhim
+  - reverse-proxy-nginx
+  - fhir-datastore-hapi-fhir
+  - monitoring
+  - database-postgres
+  - identity-access-manager-keycloak
+
+profiles:
+  - name: dev
+    packages:
+      - interoperability-layer-openhim
+      - database-postgres
+      - fhir-datastore-hapi-fhir
+      - monitoring
+      - identity-access-manager-keycloak
+      - database-postgres
+    envFiles:
+      - .env.local
+    dev: true
+    only: true
```

Comment:
Question: Do we want to only run the defined packages, which will exclude any dependant packages? I might have this wrong, but i think with this flag, only the packages specified are deployed and their dependencies are ignored